styleguide/styleguide.xml

```diff
@@ -647,6 +646,34 @@ x = temp;
 </BODY>
 </STYLEPOINT>
 
+<STYLEPOINT title="Function calls">
+<SUMMARY>Do not put a space inside the parentheses for method or function calls.</SUMMARY>
```

Comment:
This is exactly the same as the previous point.

Reply:
I may be reading that the wrong way but the previous point talked more about spaces between the name and the following parenthesis. Could be reworded to match this one though.

Reply:
Oh yeah I'm dumb

Reply:
dang it, I fixed it 5 seconds ago

Reply:
Is the new version good or should I roll it back?

Reply:
it's fine